src/MoBi/MoBi.csproj

```diff
@@ -207,6 +207,10 @@
       <Link>OSPSuite.PKParameters.xml</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="..\..\pre\libsbmlcs.dll">
+      <Link>libsbmlcs.dll</Link>
```

Comment:
what does _None_ mean in this context?
`<None Include="..\..\pre\libsbmlcs.dll">`

Compared to _Content_ in e.g.
`<Content Include="..\..\packages\OSPSuite.SimModelSolver_CVODES282.2.0.0.1\lib\net45\OSPSuite.SimModelSolver_CVODES282.dll">
`

Reply:
No idea. this is what VS is creating by default when I selected the file (that was already there in the TEst suite)

Reply:
I am going to delete the file and add it again and see if this is different